slackbot.sh

```diff
@@ -19,6 +20,11 @@ echo "SLACK_URL_RELEASE: $SLACK_URL_RELEASE"
 # Lookup the tag from the git commit hash used
 tag=$(git ls-remote "$GIT_REPO_URL" | grep "$GIT_HASH" | grep 'refs/tags/' | sed 's|.*refs/tags/||')
 
+release_id=$(curl https://api.github.com/repos/newsuk/times-public-api/releases/tags/$tag?access_token=$GIT_TOKEN | jq .id)
+
+# This also change git from pre-release to release
+changelog=$(curl --data '{"prerelease": false}' -X PATCH https://api.github.com/repos/newsuk/times-public-api/releases/$release_id?access_token=$GIT_TOKEN | jq .body | sed -e 's/"//g')
```

Comment:
Can we be very clear about this behaviour in the README?  Its potentially a little unexpected that something becomes "released" through the notification script :)

Reply:
I discussed with Stefano that it's probably best to do the release promotion though the release-documentation-cli instead.